.github/workflows/build-continual_learning.yaml

```diff
@@ -0,0 +1,24 @@
+name: Build docker images for continual learning
+run-name: Continual Learning image builder
+
+on:
+  workflow_dispatch:
+  push:
+    branches:
+      - main
```

Comment:
when this is main branch, does it mean only PR merging to main will have effect?

Reply:
Right, it will run when the main branch is updated, when a pull request edits the specified path per builder or manual trigger